src/anm/analytics.js

```diff
@@ -0,0 +1,76 @@
+var engine = require('engine');
+
+var Analytics = function () {
+    var self = this,
+        supportSendBeacon = !!navigator.sendBeacon,
+        timeout = supportSendBeacon ? 2000 : 1000,
```

Comment:
As I get it from description, the point of `sendBeacon` is that it is guaranteed to be called. So I think it's safe not to use timeouts in case of `sendBeacon`.